### ISSUE TITLE
Request new variable in form template

### ISSUE THREAD

**Issue by ahmadazimi** [kristijanhusak/laravel-form-builder#133]
Hi,

Is this possible to pass ```Form instance``` to form template?

In ```Kris\LaravelFormBuilder\Fields\FormField\Form.php``` in ```render``` method [line 660]

```php
return $this->formHelper->getView()
            ->make($this->formHelper->getConfig('form'))
            ->with(compact('showStart', 'showFields', 'showEnd'))
            ->with('formOptions', $formOptions)
            ->with('fields', $fields)
            ->with('model', $this->getModel())
            ->with('exclude', $this->exclude)
            ->with('form', $this)
            ->render();
```

**Comment by kristijanhusak** [comment]
Why you need it in the view?

**Comment by ahmadazimi** [comment]
I think this is a routine way to access instance in its template.
I want to access some functionality of Form in template (to generate complex views, for example, multi-column rows)

**Comment by kristijanhusak** [comment]
I done this on the master branch for now.

**Comment by ahmadazimi** [comment]
Thank you :+1:

**Comment by kristijanhusak** [comment]
Available in latest version(1.6.20). Thanks!
